docs/attributes-registry/system.md

```diff
@@ -0,0 +1,137 @@
+# System
+
+<!-- semconv registry.system(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `system.device` | string | The device identifier | `(identifier)` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv registry.system.cpu(omit_requirement_level) -->
```

Comment:
It would be nice if we can have sub-sections here with `## CPU Attributes`, `## Memory Attributes` and so on, like we have in the `db.md` file. That way we will have a table of contents and it's easier to find things in the doc.

Reply:
I think we should do the "grouping" in a consistent way. For `db` we solved the grouping / rendering of tech specific sections through `tags` instead of having multiple attribute groups in the registry yaml file.
I don't have a strong opinion on doing it that way or the other way, but we should keep it consistent, IMHO.

Reply:
I created an issue for this https://github.com/open-telemetry/semantic-conventions/issues/887. I think for this, we can go ahead this way.